.github/workflows/release.yml

```
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
       - uses: actions/setup-node@969bd2663942d722d85b6a8626225850c2f7be4b # tag=v3
         with:
-          node-version: 12.x
+          node-version: 18.x
```

```suggestion
          node-version: 16.x
```

Should we run CI on the **minimal** supported version instead?

---

That makes sense.

Changed the project to run on 16.